### PR TITLE
Add Chinese blog

### PR DESCRIPTION
**Pull request by xianbaoqian**
Once this is merged. I'll create another PR to create the Chinese translation under zh/

### CODE REVIEW

**Comment by xianbaoqian** [comment]
@chenglu @adeenayakup

**Comment by xianbaoqian** [comment]
Will wait until the thumbnail is ready.

**Comment by xianbaoqian** [comment]
I think we can use this page from Luke's slides https://huggingface.co/datasets/huggingface/documentation-images/blob/refs%2Fpr%2F81/blog/chinese-language-blog/announcement.png for thumbnail.

File: _blog.yml [hunk 1]
```diff
@@ -2042,4 +2042,13 @@
...
     - community
     - guide
-    - game-dev
\ No newline at end of file
+    - game-dev
+
+- local: chinese-language-blog
+  title: "Introducing HuggingFace blog for Chinese speakers: Fostering Collaboration with the Chinese AI community"
+  author: xianbao
+  thumbnail: /blog/assets/140_accelerate_transformers_with_inferentia2/thumbnail.png
```
Comment (osanseviero) [inline _blog.yml:2050]: Thumbnail link needs to be updated

File: assets/chinese-language-blog/test
Comment (osanseviero) [inline assets/chinese-language-blog/test:1]: To remove

File: assets/chinese-language-blog/wechat.jpg
Comment (osanseviero) [inline assets/chinese-language-blog/wechat.jpg:1]: All assets except thumbnail must go in https://huggingface.co/datasets/huggingface/documentation-images

File: chinese-language-blog.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+title: "Introducing HuggingFace blog for Chinese speakers: Fostering Collaboration with the Chinese AI community"
+authors:
+- user: xianbao
+- user: chenglu
```
Comment (osanseviero) [inline chinese-language-blog.md:5]:
```suggestion
- user: chenglu
  guest: true
```

**Comment by adeenayakup** [comment]
> I think we can use this page from Luke's slides https://huggingface.co/datasets/huggingface/documentation-images/blob/refs%2Fpr%2F81/blog/chinese-language-blog/announcement.png for thumbnail.

Checking with Chunte LEE, will keep you informed for the thumbnail.

**Comment by xianbaoqian** [comment]
Thanks for the review @osanseviero  PTAL :-)

**Review by osanseviero — APPROVED** [review]
Thanks! :fire:

**Comment by adeenayakup** [comment]
Here is the thumbnail made by Chunte. HF chinese blog

**Comment by xianbaoqian** [comment]
Thumbnail updated.

**Comment by xianbaoqian** [comment]
Going to merge it today if no objection.

**Pull request merged**